recipes/botan/all/conanfile.py

```diff
@@ -120,6 +120,9 @@ def _required_boost_components(self):
         return ['coroutine', 'system']
 
     def validate(self):
+        if hasattr(self, 'settings_build') and tools.cross_building(self, skip_x64_x86=True):
+            raise ConanInvalidConfiguration("Cross-building not implemented")
```

Comment:
Not implemented in the recipe or in the library? I see a mapping of Android & iOS in the recipe, so I guess that upstream build system has implemented something. But for sure it fails for iOS.

Reply:
Seems the recipe should be update with more cc-abi-flags:

```python
        if botan_compiler in ['clang', 'apple-clang', 'gcc']:
            if self.settings.arch == 'x86':
                botan_abi_flags.append('-m32')
            elif self.settings.arch == 'x86_64':
                botan_abi_flags.append('-m64')
            elif self.settings.arch in ['armv7']:
                botan_abi_flags.append('-arch armv7')
            elif self.settings.arch in ['armv8']:
                botan_abi_flags.append('-arch arm64')
```

Reply:
I confirm that I can cross compile botan to iOS arm 64 bits with this modification.